source/synthSettingsRing.py

```diff
@@ -13,17 +13,38 @@ def __init__(self,synth,setting,min=0,max=100):
 		self.min = setting.minVal if isinstance(setting, NumericDriverSetting) else min
 		self.max = setting.maxVal if isinstance(setting, NumericDriverSetting) else max
 		self.step = setting.normalStep if isinstance(setting, NumericDriverSetting) else 1
+		self.largeStep = self.setting.largeStep if isinstance(setting, NumericDriverSetting) else 10
+
+	def first(self):
+		val = self.min
+		self.value = val
+		return self._getReportValue(val)
+
+	def last(self):
+		val = self.max
+		self.value = val
+		return self._getReportValue(val)
 
 	def increase(self):
 		val = min(self.max,self.value+self.step)
 		self.value = val
 		return self._getReportValue(val)
 
+	def increase_large(self):
+		val = min(self.max, self.value + self.largeStep * 2)
+		self.value = val
+		return self._getReportValue(val)
+
 	def decrease(self):
 		val = max(self.min,self.value-self.step)
 		self.value = val
 		return self._getReportValue(val)
 
+	def decrease_large(self):

```

Comment:
Why is there an underscore in the function's name? We usually use camelCase.

```suggestion
	def decreaseLarge(self):
```